app/redux/node/reducer.ts

```diff
@@ -32,6 +34,11 @@ const reducer = (state: NodeState = initialState, action: CustomAction) => {
     }
     case LOGOUT:
       return initialState;
+    case IPC_SYNC_TYPE: {
+      if (!isMySyncChannel('config')) return state;
```

Comment:
not sure this function working here, because isMySyncChannel is a function in func, is everything is ok here?

Reply:
and have a question related to the silent exit from the invocation and return state. Maybe we want to add some message for the client.

Reply:
> and have a question related to the silent exit from the invocation and return state. Maybe we want to add some message for the client.
What do you mean?

Reply:
shouldn't we add a message for the client if we ignore if it's not a config?

Reply:
after clarification, question closed